### PR TITLE
Fix playground address checksum issue

### LINKED ISSUE #2422: Fix bug with checksum address on playground
https://discord.com/channels/707079246388133940/759631271998652466/977102133759442964

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes #2422 

Also contains lint changes

### CODE REVIEW

File: src/views/Playground.vue [hunk 1]
```diff
@@ -73,8 +75,8 @@ const scoresWithZeroBalanceAddresses = computed(() => {
     return null;
   }
   // If an address is not present inside the scoresObject, add it with a zero balance
-  const addressesArray = form.value.addresses ?? [];
-  const scoresObject = scores.value[0] ?? {};
+  const addressesArray = (form.value.addresses ?? []).map(getAddress);
+  const scoresObject = mapKeys(getAddress, scores.value[0] ?? {});
```
Comment (samuveth) [inline src/views/Playground.vue:79]: Shouldn't the the Object be the first param and the function be the second? https://www.geeksforgeeks.org/lodash-_-mapkeys-method/

Comment (ChaituVR) [reply src/views/Playground.vue:79]: Hey @samuveth this is lodash's FP module function
https://github.com/lodash/lodash/wiki/FP-Guide

It is same as normal but iteratee-first data-last methods and easy to read 😅 

Else normal lodash function will be like below (key is in second param):
```js
mapKeys(scores.value[0] ?? {}, (value, key) => getAddress(key))
```

Comment (ChaituVR) [reply src/views/Playground.vue:79]: But this is not required, should i use to normal lodash?

Comment (samuveth) [reply src/views/Playground.vue:79]: Got it :)

**Pull request merged**